Tests/Cosmos.TestRunner.Full/Cosmos.TestRunner.Full.csproj

```diff
@@ -15,7 +15,21 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Kernels\**\*.*proj" />
+        <ProjectReference Include="..\Kernels\BoxingTests\BoxingTests.csproj" />
```

Comment:
Why change this?

Reply:
I've not changed this on purpose, it is possible Visual Studio changed it when I've removed FAT2 Test? If it is so I'm unsure if makes sense that we change it again... it is a generated file.

Reply:
Project files are not really generated, and they should be minimally readable. This doesn't affect readability that much, so it's fine I think.